docs/book/src/topics/failure-domains.md

```diff
@@ -128,6 +128,21 @@ spec:
 
 ```
 
+If you can't use `Machine` (or `MachineDeployment`) to explicitly place your VMs (for example, there is a `KubeAdmControlPlane` does not accept those as an object reference but rather uses `AzureMachineTemplate` directly), then you can opt to restrict the announcement of discovered failure domains from the cluster's status itself.
```

Comment:
```suggestion
If you can't use `Machine` (or `MachineDeployment`) to explicitly place your VMs (for example, there is a `KubeadmControlPlane` does not accept those as an object reference but rather uses `AzureMachineTemplate` directly), then you can opt to restrict the announcement of discovered failure domains from the cluster's status itself.
```

Reply:
> for example, there is a `KubeadmControlPlane` does not accept those as an object reference 

I think we're missing a word here; I couldn't quite parse this paragraph...

Reply:
fixed, sorry